spec.html

```diff
@@ -5871,6 +5871,29 @@ <h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-sametype" type="abstract operation">
+      <h1>
+        SameType (
+          _x_: an ECMAScript language value,
+          _y_: an ECMAScript language value,
+        ): a Boolean
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>It determines whether or not the two arguments are the same type.</dd>
+      </dl>
+      <emu-alg>
+        1. If _x_ is _y_, return *true*.
```

Comment:
I actually think this should go back. See https://github.com/tc39/ecma262/pull/3408#discussion_r1735371721.